### DISCUSSION TITLE
Issue in installing cocoapods on M1

### DISCUSSION THREAD

**Discussion by dkchrome** [Homebrew/discussions#3982] [Linux]
Hi,

I am trying to install the cocoapods on M1 MacBook using Brew via terminal with following command:  

"brew install cocoapods"

But I am getting following error on terminal:

==> Downloading https://ghcr.io/v2/homebrew/core/ca-certificates/blobs/sha256:1b264e579e31b3041a87ff91f09d5f7cc0d51fea1c83e63aee17a1b95509cbe
curl: (22) The requested URL returned error: 503                              
Warning: Problem : HTTP error. Will retry in 1 seconds. 3 retries left.
curl: (22) The requested URL returned error: 503                              
Warning: Problem : HTTP error. Will retry in 2 seconds. 2 retries left.
curl: (22) The requested URL returned error: 503                              
Warning: Problem : HTTP error. Will retry in 4 seconds. 1 retries left.
curl: (22) The requested URL returned error: 503                              

Error: cocoapods: Failed to download resource "ca-certificates"
Download failed: https://ghcr.io/v2/homebrew/core/ca-certificates/blobs/sha256:1b264e579e31b3041a87ff91f09d5f7cc0d51fea1c83e63aee17a1b95509cbe1

=============================================

When visit the above link, it says following error:   "No server is currently available to service your request."

Please suggest what is the error, or how can correctly install the cocoapods?

Thanks

Update: Often getting this error:

==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:1b264e579e31b3041a87ff91f09d5f7cc0d51fea1c83e63aee17a1b9
curl: (22) The requested URL returned error: 403                              

Error: cocoapods: Failed to download resource "ca-certificates"
Download failed: https://ghcr.io/v2/homebrew/core/ca-certificates/blobs/sha256:1b264e579e31b3041a87ff91f09d5f7cc0d51fea1c83e63aee17a1b95509cbe1

**Comment by Bo98** [comment]
This is a GitHub outage. Follow https://www.githubstatus.com/incidents/ljwdk6qml8z8 for updates.
